### DISCUSSION TITLE
Grafana-data rangeUtil meant as a package outside of Grafana?

### DISCUSSION THREAD

**Discussion by hoetmaaiers** [grafana/grafana#67659] [Alerting]
Is the grafana-data [rangeUtil.ts](https://github.com/grafana/grafana/blob/main/packages/grafana-data/src/datetime/rangeutil.ts) meant to be used as a package outside of Grafana? 

Since I can't find any documentation about these functions I am in doubt. 

If the TimeRange notation (e.g. `now-1h`) is Grafana specific, I will want to use grafana-data from handling this. Or maybe the format is more standardized, but I can't find this. Any thoughts?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). This is phrased as a question, and we'd like questions about grafana to go to our[ community forum](https://community.grafana.com/). Feel free to make an [issue](https://github.com/grafana/grafana/issues) if you cannot find a solution and would like to request the feature.
